client/src/components/Panel.js

```diff
@@ -91,6 +91,11 @@ export class Panel extends React.Component {
                 className={classNames("panel-heading-utils")}
                 onClick={() => this.setState({ isOpen: !isOpen })}
                 aria-labelledby={label_id}
+                aria-label={
```

Comment:
Oh, wait, shouldn't have both a labelled by and a label. What element is `label_id` supposed to refer to?

Reply:
oh yeah this is actually fixed in a11y_review branch #800 ! I have to admit It's getting a bit confusing because I have branches unnecessarily merging into another (unintentionally because I was waiting for PR reviews!)